### PR TITLE
Update copy for GenBank switch

### PR DESCRIPTION
**Pull request by danrlu**
### Summary:
Based on [to-do list](https://docs.google.com/document/d/1m4LgaxJM8qg8iMupN9HKv0lue7B-Z6fMGt9pLBdpYqA/edit#heading=h.qijild2ouu86) from discussion with Karyna. Even if we need to switch back to GISAID, all the changes here can stay except for instructions for tree build sample ID box [here](https://github.com/chanzuckerberg/czgenepi/blob/trunk/src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/CreateNSTreeModal/components/SampleIdInput/components/InputInstructions/strings.ts).

### Demos:
Need to push to staging to check.

### Checklist:
- [x] I merged latest `<base branch>`
- [x] I manually verified the change
- [x] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/CreateNSTreeModal/components/SampleIdInput/index.tsx [hunk 1]
```diff
@@ -155,7 +155,7 @@ const SampleIdInput = ({
         variant="outlined"
         rows={!isInEditMode ? 4 : 3}
         value={isInEditMode ? inputValue : inputDisplayValue}
-        placeholder="e.g. U12345, AF123456"
```
Comment (danrlu) [inline src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/CreateNSTreeModal/components/SampleIdInput/index.tsx:158]: this is the examples in the tree build force include sample box. It's shared between SARS-CoV-2 build and mpox. SARS-CoV-2 need `USA/CA-CZB-0000/2021` and mpox need `AF123456`

File: src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/columnDefinitions/sc2PublicId.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const sc2PublicIdColumn: ColumnDef<Sample, any> = {
       tooltipStrings={{
         boldText: "Public ID",
         regularText:
-          "Accession ID from public repository or public ID generated by CZ Gen Epi",
```
Comment (danrlu) [inline src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/columnDefinitions/sc2PublicId.tsx:16]: GenBank has 2 IDs, `Isolate name` and `Accession` so specify which one we needs here.

File: src/frontend/src/components/DownloadMetadataTemplate/common/constants.ts [hunk 3]
```diff
@@ -22,7 +22,7 @@ const GENERAL_VIRAL_SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS: Record<
 > = {
   ...BASE_METADATA_HEADERS,
   privateId: "Private ID",
-  publicId: "Genbank Accession (Public ID)" + OPTIONAL_HEADER_MARKER,
```
Comment (danrlu) [inline src/frontend/src/components/DownloadMetadataTemplate/common/constants.ts:25]: this is for mpox upload. we only ask for genbank accessions

File: src/frontend/src/components/DownloadMetadataTemplate/common/constants.ts [hunk 4]
```diff
@@ -114,7 +114,7 @@ export const UPLOAD_EXAMPLE_ROWS: PathogenConfigType<string[][]> = {
     [
       EXAMPLE_SAMPLE_IDS[0], // sampleId
       "Private sample name", // privateId
-      "(if available) GISAID ID", // publicId -- here as explainer
```
Comment (danrlu) [inline src/frontend/src/components/DownloadMetadataTemplate/common/constants.ts:117]: for SARS-CoV-2, we accept both types of IDs

File: src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/CreateNSTreeModal/components/SampleIdInput/components/InputInstructions/strings.ts [hunk 5]
```diff
@@ -9,9 +9,9 @@ interface InputInstructionsStrings {
 export const InputInstructionsPathogenStrings: PathogenConfigType<InputInstructionsStrings> =
   {
     [Pathogen.COVID]: {
-      publicRepositoryIdType: "GenBank strain names",
```
Comment (danrlu) [inline src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/CreateNSTreeModal/components/SampleIdInput/components/InputInstructions/strings.ts:12]: tree build sample ID input box instructions

File: src/frontend/src/components/DownloadMetadataTemplate/common/constants.ts [hunk 6]
```diff
@@ -22,7 +22,7 @@ const GENERAL_VIRAL_SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS: Record<
 > = {
   ...BASE_METADATA_HEADERS,
   privateId: "Private ID",
-  publicId: "Genbank Accession (Public ID)" + OPTIONAL_HEADER_MARKER,
+  publicId: "GenBank Accession (Public ID)" + OPTIONAL_HEADER_MARKER,
```
Comment (danrlu) [inline src/frontend/src/components/DownloadMetadataTemplate/common/constants.ts:25]: this is for mpox upload. we only ask for genbank accessions

File: src/frontend/src/components/DownloadMetadataTemplate/common/constants.ts [hunk 7]
```diff
@@ -114,7 +114,7 @@ export const UPLOAD_EXAMPLE_ROWS: PathogenConfigType<string[][]> = {
     [
       EXAMPLE_SAMPLE_IDS[0], // sampleId
       "Private sample name", // privateId
-      "(if available) GISAID ID", // publicId -- here as explainer
+      "(if available) GenBank Isolate Name or GISAID ID", // publicId -- here as explainer
```
Comment (danrlu) [inline src/frontend/src/components/DownloadMetadataTemplate/common/constants.ts:117]: for SARS-CoV-2, we accept both types of IDs

File: src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/CreateNSTreeModal/components/SampleIdInput/components/InputInstructions/strings.ts [hunk 8]
```diff
@@ -9,9 +9,9 @@ interface InputInstructionsStrings {
...
+      publicRepositoryIdType: "GenBank Isolate Names without SARS-CoV-2/human/ prefix",
```
Comment (danrlu) [inline src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/CreateNSTreeModal/components/SampleIdInput/components/InputInstructions/strings.ts]: tree build sample ID input box instructions

File: src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/CreateNSTreeModal/components/SampleIdInput/index.tsx [hunk 9]
```diff
@@ -155,7 +155,7 @@ const SampleIdInput = ({
...
+        placeholder="e.g. USA/CA-CZB-0000/2021, AF123456"
```
Comment (danrlu) [inline src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/CreateNSTreeModal/components/SampleIdInput/index.tsx:158]: this is the examples in the tree build force-include sample box. It's shared between SARS-CoV-2 build and mpox. SARS-CoV-2 need `USA/CA-CZB-0000/2021` and mpox need `AF123456`

File: src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/columnDefinitions/sc2PublicId.tsx [hunk 10]
```diff
@@ -13,7 +13,7 @@ export const sc2PublicIdColumn: ColumnDef<Sample, any> = {
...
+          "GenBank Isolate Name, GISAID ID or public ID generated by CZ Gen Epi",
```
Comment (danrlu) [inline src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/columnDefinitions/sc2PublicId.tsx:16]: GenBank has 2 types of IDs, Isolate name and Accession so specify which one it is here.

**Comment by danrlu** [comment]
GISAID test is failing. Could someone help me force merge?

**Review by jgadling — APPROVED** [review]

**Pull request merged**